diff --git a/google_map_field.module b/google_map_field.module
index 381036c..be99c46 100644
--- a/google_map_field.module
+++ b/google_map_field.module
@@ -188,7 +188,7 @@ function google_map_field_format_field($item, $delta, $display) {
  */
 function google_map_field_field_widget_form(&$form, &$form_state, $field, $instance, $langcode, $items, $delta, $element) {
 
-  $form['#attached']['js'][] = google_map_field_get_protocol() . 'maps.googleapis.com/maps/api/js?sensor=false';
+  $form['#attached']['js'][] = google_map_field_get_protocol() . 'maps.googleapis.com/maps/api/js?sensor=false&key=' . variable_get('google_map_field_apikey', '');
 
   $field_id = is_a($element['#entity'], 'FieldCollectionItemEntity') ? $element['#entity']->item_id : $field['id'];
   $field_id = empty($field_id) ? rand(10000000, 99999999) : $field_id;
